File: program/find-area-of-a-cuboid/find_area_of_a_cuboid.rs
fn main() {
    let length = 2.0;
    let width = 3.0;
    let height = 4.0;
    let area = 2.0 * (length * width + length * height + width * height);

    println!("Area of cuboid is {0}", area as f32);
}

File: program/find-area-of-a-cuboid/find_area_of_a_cuboid.dart
import 'dart:io';
void main(){
  print("Enter the length :");
  int length = int.parse(stdin.readLineSync()!);
  print("Enter the width :");
  int width = int.parse(stdin.readLineSync()!);
  print("Enter the height :");
  int height = int.parse(stdin.readLineSync()!);
  int cuboid = 2 * (length * width + length * height + width * height);
  print('Area of cuboid : $cuboid');
}
